docs/src/modules/components/AppNavDrawerItem.js

```diff
@@ -32,9 +30,7 @@ const iconsMap = {
   AddIcon: AddCircleRoundedIcon,
   BookIcon: BookRoundedIcon,
   ReaderIcon: ChromeReaderModeRoundedIcon,
-  TableViewIcon: TableViewRoundedIcon,
   ExperimentIcon: ScienceIcon,
-  DatePickerIcon: DateRangeRounded,
```

Comment:
These icons are only used in MUI X, so I will pass them through `IconComponent` instead.